airflow/serialization/serialized_objects.py

```diff
@@ -165,7 +165,11 @@ def __init__(self, type_string: str) -> None:
         self.type_string = type_string
 
     def __str__(self) -> str:
-        return f"Timetable class {self.type_string!r} is not registered"
+        return (
+            f"Timetable class {self.type_string!r} is not registered or "
+            "you have a top level database access that disrupted the session. "
+            "Please check the airflow best practices documentation."
+        )
```

Comment:
I sort of feel that mentioning _top level database access_ is a bit too specific. Maybe it’d be better to be slightly more vague here (there’s something wrong with eager variable evaluation in the class) instead. But I’m not strong on this—we can always reword this afterwards.